plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -155,9 +155,9 @@ class ServicesTable extends React.Component {
       return null;
     }
 
-    return (
+    const webLink = (
```

Comment:
I don't see us using the `webLink` variable anywhere I'd assume this is a forgotten change?

Reply:
good catch… it actually was a leftover from a test I did - should have been reverted. will fix it